Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -169,6 +170,55 @@ function Run_IMB_MPI1() {
 	fi
 }
 
+function Run_OSU_P2P() {
+	total_attempts=$(seq 1 1 $osu_p2p_tests_iterations)
+	if [[ $osu_p2p_tests == "all" ]]; then
+		extra_params=""
+	else
+		extra_params=$osu_p2p_tests
+	fi
+	for attempt in $total_attempts; do
+		LogMsg "OSU P2P test iteration $attempt for $mpi_type- Running."
+		case "$mpi_type" in
+			ibm)
+				LogMsg "Test not yet available"
+			;;
+			open)
+				LogMsg "Test not yet available"
+			;;
+			hpcx)
+				LogMsg "Test not yet available"
+			;;
+			intel)
+				LogMsg "Test not yet available"
+			;;
+			mvapich)
+				LogMsg "$osu_p2p_tests for $mpi_type"
+				LogMsg "$mpi_run_path -n $(($mpi1_ppn * $total_virtual_machines)) $master $slaves_array $mpi_settings $osu_p2p_path $extra_params"
+				ssh root@${master} "$mpi_run_path -n $(($mpi1_ppn * $total_virtual_machines)) $master $slaves_array $mpi_settings $osu_p2p_path$extra_params> OSU-P2P-AllNodes-output-Attempt-${attempt}.txt"
```

Comment:
Adding process binding will be useful to compare performance results: (Or make this as a parameter to be passed in)

Reply:
Added "INFINIBAND_OMB_MVAPICH_MPI_SETTINGS" for mpi settings.  The default value is MV2_CPU_BINDING_LEVEL=numanode MV2_CPU_BINDING_POLICY=scatter . This parameter can be customized when running the test.